tests/robot-cases/Group11-Nightly/Replication.robot

```diff
@@ -130,8 +130,24 @@ Test Case - Trigger Manual
     Page Should Contain  hello-world
     Go Into Repo  hello-world
     Page Should Contain  latest
-    Close browser
+    Close Browser
 
+Test Case - Large Image Replicate
+    Init Chrome Driver
+    ${d}=    Get Current Date  result_format=%m%s
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    Create An New Project  project${d}
+    Push Image with tag  ${ip}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}  project${d}  ubuntu  16.04  16.04
+    Switch To Replication Manage
+    Create A Rule With Existing Endpoint  rule${d}  project${d}  edp  Immediate
+    Logout Harbor
```

Comment:
Let's add one more step "Close Brower" here to ensure that the case is visiting the target endpoint from then.